.github/workflows/build-and-test-toolchain.yml

```diff
@@ -141,13 +141,14 @@ jobs:
       - name: Execute GCC tests
         run: |
           cd ~/work
-          .github/scripts/toolchain/execute-gcc-tests.sh "${{ env.TAG }}" "${{ env.MODULE }}" "${{ env.FILTER }}"
+          .github/scripts/toolchain/execute-gcc-tests.sh $ARTIFACT_PATH/gcc-tests-${{ env.TAG }} \
```

Comment:
could it be refactored with ~/work/github... without `cd` call?

Reply:
OK, I'll do a generic refactor in a separate PR.